README.md

```diff
@@ -72,10 +72,9 @@ Watson services are migrating to token-based Identity and Access Management (IAM
 ### Getting credentials
 To find out which authentication to use, view the service credentials. You find the service credentials for authentication the same way for all Watson services:
 
-1.  Go to the IBM Cloud **[Dashboard][watson-dashboard]** page.
-2.  Either click an existing Watson service instance or click **Create**.
-3.  Click **Show** to view your service credentials.
-4.  Copy the `url` and either `apikey` or `username` and `password`.
+1. Go to the IBM Cloud [Dashboard](https://console.bluemix.net/dashboard/apps?category=ai) page.
```

Comment:
Since this previously used a variable for the url, you should either keep the use of the variable and redefine the url at the bottom of the file, or keep the new changes and just delete the unused url at the bottom

Reply:
👍